### ISSUE TITLE
Type of TimeSync.serverTime() with fixed date

### ISSUE THREAD

**Issue by steph643** [mizzao/meteor-timesync#23]
Here is what I get:
```javascript
console.log(typeof TimeSync.serverTime(new Date));
console.log(typeof TimeSync.serverTime());
```
```
string
number
```
Am I not supposed to get a Date?

(dammit, I keep forgetting I am a programmer!)

**Comment by mizzao** [comment]
The first one seems to be a bug, I'll look into it.

The second is a unix epoch, which you can convert into a Date with `new Date(TimeSync.serverTime(...))`

**Comment by steph643** [comment]
Thanks a lot.
In my opinion, there is no need for serverTime() to return a Date object as long as you put your explanation in the doc.

**Comment by mizzao** [comment]
This is now fixed in the (just released) 0.3.1.

**Comment by steph643** [comment]
Works well, thanks!
